source/_includes/navigation.html

```diff
@@ -28,7 +28,7 @@
 		<li>
 			<a href="#">Posts</a>
 			<ul class="dl-submenu">
-				<li><a href="/posts/">All Posts</a></li>
+				<li><a href="/archives/">Archives</a></li>
```

Comment:
It looks like there's no real difference between the `/posts/` and `/archives/` routes. Would you be able make the `/posts/` route display the same thing as the new `/archives/` route (perhaps through a simple `include` statement)?